corehq/apps/repeaters/dbaccessors.py

```diff
@@ -70,23 +70,23 @@ def get_paged_repeat_records(domain, skip, limit, repeater_id=None, state=None):
         'reduce': False,
         'limit': limit,
         'skip': skip,
+        'descending': True,
     }
 
     if repeater_id and not state:
-        kwargs['startkey'] = [domain, repeater_id]
-        kwargs['endkey'] = [domain, repeater_id, {}]
+        kwargs['endkey'] = [domain, repeater_id]
+        kwargs['startkey'] = [domain, repeater_id, {}]
```

Comment:
just found this: https://developer.couchbase.com/documentation/mobile/current/guides/couchbase-lite/native-api/query/index.html#querying-key-ranges. it may be a better explanation than i just gave